code/datums/wounds/bones.dm

```diff
@@ -0,0 +1,282 @@
+		if(25 to 50)
+			splint_condition = "loosely "
+		if(50 to 75)
+			splint_condition = "mostly "
+		if(75 to INFINITY)
+			splint_condition = "tightly "
+	return "<B>[victim.p_their(TRUE)] [limb.name] is [splint_condition] fastened in a splint of [splinted.name]!</B>"
+
+/*
+	New common procs for /datum/wound/brute/bone/
+*/
+
+/datum/wound/brute/bone/proc/update_inefficiencies()
+	if(limb.body_zone in list(BODY_ZONE_L_LEG, BODY_ZONE_R_LEG))
+		if(splinted)
+			limp_slowdown = initial(limp_slowdown) * splinted.splint_factor
+		else
+			limp_slowdown = initial(limp_slowdown)
+		victim.apply_status_effect(STATUS_EFFECT_LIMP)
+	else if(limb.body_zone in list(BODY_ZONE_L_ARM, BODY_ZONE_R_ARM))
+		if(splinted)
+			interaction_efficiency_penalty = 1 + ((interaction_efficiency_penalty - 1) * splinted.splint_factor)
+		else
+			interaction_efficiency_penalty = interaction_efficiency_penalty
+
+	if(initial(disabling) && splinted)
+		disabling = FALSE
+		REMOVE_TRAIT(limb, TRAIT_LIMB_DISABLED_WOUND, src)
+	else if(initial(disabling))
+		ADD_TRAIT(limb, TRAIT_LIMB_DISABLED_WOUND, src)
+		disabling = TRUE
+
+	limb.update_wounds()
+
+/*
+	BEWARE OF REDUNDANCY AHEAD THAT I MUST PARE DOWN
+*/
+
+/datum/wound/brute/bone/proc/splint(obj/item/stack/I, mob/user)
+	if(splinted && splinted.splint_factor >= I.splint_factor)
+		to_chat(user, "<span class='warning'>The splint already on [user == victim ? "your" : "[victim]'s"] [limb.name] is better than you can do with [I].</span>")
+		return
+
+	user.visible_message("<span class='danger'>[user] begins splinting [victim.p_their()] [limb.name] with [I].</span>", "<span class='warning'>You begin splinting [user == victim ? "your" : "[victim]'s"] [limb.name] with [I]...</span>")
+
+	if(!do_after(user, base_treat_time * (user == victim ? 1.5 : 1), target = victim, extra_checks=CALLBACK(src, .proc/still_exists)))
+		return
+
+	user.visible_message("<span class='green'>[user] finishes splinting [victim]'s [limb.name]!</span>", "<span class='green'>You finish splinting [user == victim ? "your" : "[victim]'s"] [limb.name]!</span>")
+	treat_priority = FALSE
+	splinted = new I.type(limb)
+	splinted.amount = 1
+	I.use(1)
+	update_inefficiencies()
+
+/*
+	Moderate (Joint Dislocation)
+*/
+
+/datum/wound/brute/bone/moderate
+	name = "Joint Dislocation"
+	desc = "Patient's bone has been unset from socket, causing pain and reduced motor function."
+	treat_text = "Recommended application of bonesetter to affected limb, though manual relocation by applying an aggressive grab to the patient and helpfully interacting with afflicted limb may suffice."
+	examine_desc = "is awkwardly jammed out of place"
+	occur_text = "jerks violently and becomes unseated"
+	severity = WOUND_SEVERITY_MODERATE
+	viable_zones = list(BODY_ZONE_L_ARM, BODY_ZONE_R_ARM, BODY_ZONE_L_LEG, BODY_ZONE_R_LEG)
+	interaction_efficiency_penalty = 1.5
+	limp_slowdown = 3
+	threshold_minimum = 35
+	threshold_penalty = 15
+	treatable_tool = TOOL_BONESET
```

Comment:
can you add a ghetto way to reset a dislocated limb that doesn't involve another person?

even the option to splint a dislocated joint with gauze or tape or something would be nice